Updates to Create a Cultural protocol (#396)
MukurtuCMS/mukurtu-user-manual: docs/communities-cultural-protocols-categories/CreateACulturalProtocol.md

Reviewer: ```suggestion
This article covers two other methods for creating a cultural protocol: from the dashboard, and from a community page.
```

Reviewer: Don't need these first two screenshots.

Reviewer: Roles in bold.

Reviewer: ```suggestion
    In order of increasing responsibility, protocol user roles are: 
```

    ![The add communities modal, with all available communities displayed. Two communities are selected, indicated by a green outline. The add communities button is located at the bottom left.](../_embeds/create-protocol-05.png) 

5. Add an optional description. Descriptions display on the protocol page and may include information about the purpose of the protocol, the type of content under this protocol, and who to contact if you have qustions.

    ![The description field](../_embeds/create-protocol-06.png)   

6. Select the *membership display setting*. Choose from one of: 

    - **Do not display** does not display any protocol members on the protocol page.
    - **Display protocol stewards** displays protocol stewards and not protocol members on the community page.
    - **Display all members** displays all members on the protocol page.

    ![The membership display field](../_embeds/create-protocol-07.png)

    At this point you can add community members to the new protocol. Protocol role descriptions are availble and can be expanded as a reference. 

    ![A list of protocol role descriptions](../_embeds/create-protocol-23.png)

>     In order of responsibilitiy, protocol user roles are: 

Reviewer: Bold role names

>     In order of responsibilitiy, protocol user roles are: 

Reviewer: ```suggestion
    In order of increasing responsibility, protocol user roles are: 
```

8. Select the user you want to add, and select "Add" to the right of the search bar. The user will appear in the table below. Repeat steps 1 and 2 to add additional members.

    ![The selected user in the search bar with the add button highlighted](../_embeds/create-community-11.png)

9. Select one or more user roles to for each new member by selecting the corresponding checkboxes. Select "Remove" to remove users from the group. 

    ![The new member table with user roles checked. The remove box is highlighted](../_embeds/create-community-15.png)

    !!! requirement
        All users must be assigned at least one protocol role or the form will not save.
        
    By default, the user that creates the protocol will be added to the protocol and assigned the protocol steward role. The protocol steward role cannot be removed from the creator in this form.

10. When you are finished, select "Save and create another protocol" to generate a fresh form, or "Save" to return to the community page.

11. The community page will load with a success message. 

>     ![The community page with a success message indicating that the protocol was successfully created.](../_embeds/create-protocol-09.png)

Reviewer: Is this the right screenshot?

Author: Whoops. Wrong file name. That should be fixed now.

> ## Create a cultural protocol through an existing community

Reviewer: ```suggestion
## Create a cultural protocol from a community page
```
